_commands/tools/grep.md

```diff
@@ -3,22 +3,100 @@
 
 grep
 -------
-TODO: Add documentation for this command by submitting a pull request.
-<!-- one line explanation would go here -->
 
-<!-- minimal example -->
+`grep` is a program for searching text files for lines that match regular exprssions. It can be used for all sorts of pattern-matching and text-based query analysis.
+
```

Comment:
Include a short example here so it's displayed on the reference page. Such as your first example down below.

Reply:
I had pushed the first example because it was too long for the home page. I added in a newer, singular example that is shorter and fits nicely on the menu page.